config.json

```
@@ -810,6 +810,16 @@
         ],
         "difficulty": 4
       },
+      {
+        "slug": "rate-limiter",
+        "name": "rate-limiter",
```

Name shouldn't be kebab case.

```suggestion
        "name": "Rate Limiter",
```